nodejs/awsx/ec2/natGateway.ts

```diff
@@ -31,7 +31,7 @@ export class NatGateway
     constructor(name: string, vpc: x.ec2.Vpc, args: NatGatewayArgs | ExistingNatGatewayArgs, opts: pulumi.ComponentResourceOptions = {}) {
         super("awsx:x:ec2:NatGateway", name, {}, { parent: vpc, ...opts });
 
-        const parentOpts = { parent: this };
+        const parentOpts = { parent: vpc };

```

Comment:
this doesn't seem right.  we'd want the child resources to see this NatGateway as their parent.  is there a reason this needed ot happen?

Reply:
Fixed